src/components/Select/Select.component.tsx

```diff
@@ -64,157 +64,169 @@ import {
+    useEffect(() => {
+        document.body.addEventListener('click', onDropDownCloseHandler);
 
-		return () => {
-			document.body.removeEventListener('click', onDropDownCloseHandler);
-		};
-	}, []);
+        return () => {
+            document.body.removeEventListener('click', onDropDownCloseHandler);
+        };
+    }, []);
 
-	/**
+    /**
 	 * If 'isOpen' is true and 'value' is empty, then set the active index to 0 and scroll the list to (0,0)
 	 * else, set the active index to selected item index
 	 */
-	useEffect(() => {
-		if (isOpen) {
-			if (value === '') {
-				setActive(0);
-				listRef.current.scrollTo(0, 0);
-			} else setActive(selectedIndex);
-		}
-	}, [isOpen]);
+    useEffect(() => {
+        if (isOpen) {
+            if (value === '') {
+                setActive(0);
+                listRef.current.scrollTo(0, 0);
+            } else {
+                setActive(selectedIndex);
+            }
+        }
+    }, [isOpen]);
 
-	return (
-		<Wrapper>
-			<Input
-				ref={inputRef}
-				placeholder=' '
-				type='text'
-				onFocus={onFocusHandler}
-				readOnly={true}
-				onKeyDown={onKeyDown}
-				value={value}
-				leadingIcon={leadingIcon}
-			/>
-			<Label leadingIcon={leadingIcon}>{label}</Label>
-			{leadingIcon && (
-				<LeadingIcon className='select__leading-icon'>
-					<Icon name={leadingIcon} size={16}/>
-				</LeadingIcon>
-			)}
-			<TrailingIcon
-				className='select__trailing-icon'
-				ref={trailingIconRef}
-				onClick={onIconTrailingIconClickHandler}
-			>
-				<Icon name={(isOpen || value) ? 'Close' : 'ArrowDown'} size={16}/>
-			</TrailingIcon>
-			{options.length > 0 && (
-				<List
-					ref={listRef}
-					isOpen={isOpen}
-					listItems={options}
-					handleItemClick={onUserSelectHandler}
-					value={value}
-					activeItem={active}
-				/>
-			)}
-		</Wrapper>
-	);
+    return (
+        <Wrapper>
+            <Input
+                ref={inputRef}
+                placeholder=' '
+                type='text'
+                onFocus={onFocusHandler}
+                readOnly={true}
+                onKeyDown={onKeyDown}
+                value={value}
+                leadingIcon={leadingIcon}
+            />
+            <Label leadingIcon={leadingIcon}>{label}</Label>
+            {leadingIcon && (
+                <LeadingIcon className='select__leading-icon'>
+                    <Icon
+                        name={leadingIcon}
+                        size={16}
+                    />
+                </LeadingIcon>
+            )}
+            <TrailingIcon
+                className='select__trailing-icon'
+                ref={trailingIconRef}
+                onClick={onIconTrailingIconClickHandler}
+            >
+                <Icon
+                    name={(isOpen || value) ? 'Close' : 'ArrowDown'}
+                    size={16}
+                />
+            </TrailingIcon>
+            {options.length > 0 && (
```

Comment:
```suggestion
            {options.length && (
```

Reply:
this will change the ts type. And can also return 0